tasks/us/census/acs.py

```diff
@@ -24,6 +24,14 @@
 from time import time
 LOGGER = get_logger(__name__)
 
+GEOGRAPHIES = ['state', 'county', 'census_tract', 'block_group',
+                        'puma', 'zcta5', 'school_district_elementary',
+                        'congressional_district',
+                        'school_district_secondary',
+                        'school_district_unified', 'cbsa', 'place']
```

Comment:
The alignment is a bit off here. Shouldn't everything be aligned so that continuation lines start at the same level as `'state'`?

Reply:
Blame the linter, but you are right, I'm changing it

Reply:
f*** linter!